src/app/views/public/landing/landing.html

```diff
@@ -0,0 +1,82 @@
+
+<div id="public">
+  <nav class="navbar navbar-default navbar-fixed-top">
```

Comment:
You need to create a `layout.html` for the public pages common html. And then use `ui-view` to display the current view. (cf. Onboarding)